content/docs/use-cases/ml-model-registries.md

```diff
@@ -0,0 +1,54 @@
+# Machine Learning Model Registries
+
+An **ML model registry** (or _model catalog_) works as a central repository
+between data science projects and their applications. ML models and other data
+artifacts can be browsed, shared, audited, tested, and deployed from here. DVC
+repositories support all of this while integrating seamlessly with existing
+development workflows. Other tools from [Iterative](https://iterative.ai/)
+further improve your model catalogs with more features and better usability.
+
+![](/img/ml_model_registry_placement.jpg) _Model registry placement contrasts
+with general [data registries] for reusable data sources or training sets._
+
+<!-- TODO: Look up package registry diagrams for inspiration -->
+
+[data registries]: /doc/use-cases/data-registries
+
+DVC model registries provide your team with key collaborative capabilities:
+
+- Store, organize, and track the versions of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to early-stage [experiments] and production-ready
+  models alike.
+- Save performance [metrics and plots] to quickly evaluate and compare models.
+- A platform to export, integrate, deploy, or roll-back published models
+
+[experiments]: /doc/user-guide/experiment-management
+[metrics and plots]: /doc/start/metrics-parameters-plots
+
+Many of these benefits are built-into DVC: Your [data pipelines] and resulting
```

Comment:
can we do pipelines as second thing in the list or even omit (make it less DVC-specific), "how model is built", "its metrics", etc

Reply:
> do pipelines as second thing ... (make it less DVC-specific 

Agreed. Updated to "modeling process and performance metadata" (term "metrics and plots" is already used  earlier).